src/Farmer/Builders/Builders.Vm.fs

```diff
@@ -12,6 +12,10 @@ open System
 
 let makeName (vmName:ResourceName) elementType = ResourceName $"{vmName.Value}-%s{elementType}"
 
+type SshObject = {
```

Comment:
This probably belongs in the Farmer/Arm/Compute.fs file.

Reply:
ended up just removing it